recipes/turbo/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "turbo" %}
+{% set version = "v0.6.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gottingen/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: eda48c6fea7286349cb7484b04cdf66a5583aac31cf5d4f25bee8192970936ec
+
+build:
```

Comment:
This is a shared library and should thus also have a run_exports entry here.

Reply:
yet, the version has change nearly one  hundred times, it is 0.9.41 now, how should I do

Reply:
Then add a run_exports for x.x.x